src/app/(admin)/admin/hints/actions.tsx

```diff
@@ -45,7 +46,10 @@ export async function claimHint(hintId: number) {
 
   if (hint.claimer && hint.claimer !== session.user.id) {
     revalidatePath("/admin/");
-    throw new Error("Hint claimed by " + hint.claimer + ".");
+    const user = await db.query.teams.findFirst({
```

Comment:
Might also want to avoid this query if there's a faster way to find claimer display name